changelog/6240.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Log catching mechanism creates LogCaptureHandler if necessary.
```

Comment:
I agree, reader of the changelog doesn't really need to know this, so would remove this line.

Reply:
Done